src/edu/stanford/nlp/sempre/Parser.java

```diff
@@ -56,6 +56,9 @@
 
     @Option(gloss = "Dump all features (for debugging)")
     public boolean dumpAllFeatures = false;
+
+    @Option(gloss = "Whether rules are marked as floating by default")
+    public boolean defaultIsFloating = true;
```

Comment:
Sure.  Document that this is only used by the individual parsers, and it doesn't affect the beam parser.